Sources/PDFXKitObjC/Private/PDFXKit+PSPDFKit.h

```diff
@@ -0,0 +1,24 @@
+//
+//  Copyright (c) 2017-2020 PSPDFKit GmbH. All rights reserved.
+//
+//  The PSPDFKit Sample applications are licensed with a modified BSD license.
+//  Please see License for details. This notice may not be removed from
+//  this file.
+//
+
+#import "PDFXAction+PSPDFKit.h"
+#import "PDFXActionGoTo+PSPDFKit.h"
+#import "PDFXActionNamed+PSPDFKit.h"
+#import "PDFXActionResetForm+PSPDFKit.h"
+#import "PDFXActionRemoteGoTo+PSPDFKit.h"
+#import "PDFXActionURL+PSPDFKit.h"
+#import "PDFXAnnotation+PSPDFKit.h"
```

Comment:
The original one also skipped this declaration but we should it back.

```suggestion
#import "PDFXAnnotation+PSPDFKit.h"
#import "PDFXAnnotationPopup+PSPDFKit.h"
```